aries-site/src/pages/components/index.js

```diff
@@ -4,53 +4,55 @@ import { Box, Heading, Paragraph } from 'grommet';
 
 import { CardGrid, ContentCard, Meta } from '../../components';
 import { Layout, PageIntro } from '../../layouts';
-import { getCards, getPageDetails, nameToPath, useDarkMode } from '../../utils';
+import { getCards, getPageDetails, nameToPath } from '../../utils';
 
 const title = 'Components';
 const pageDetails = getPageDetails(title);
 const cards = getCards(title);
 
-const Components = () => {
-  const darkMode = useDarkMode();
-  const cardImage = darkMode.value
-    ? '/carte-components-dark.svg'
-    : '/carte-components-light.svg';
-
-  return (
-    <Layout title={title} isLanding>
-      <Meta
-        title={title}
-        description={pageDetails.seoDescription}
-        canonicalUrl="https://design-system.hpe.design/components"
-      />
-      <Box gap="large">
-        <PageIntro
-          image={{
-            src: cardImage,
-            alt: 'Card deck of HPE Design System component cards',
-            fit: 'cover',
-          }}
-        >
-          <Box justify="center" fill>
-            <Heading margin="none">{title}</Heading>
-            <Paragraph fill>{pageDetails.description}</Paragraph>
-          </Box>
-        </PageIntro>
-        <CardGrid>
-          {cards.map(topic => (
-            // Need to pass href because of: https://github.com/zeit/next.js/#forcing-the-link-to-expose-href-to-its-child
-            <Link key={topic.name} href={nameToPath(topic.name)} passHref>
-              <ContentCard
-                as="a"
-                style={{ textDecoration: 'none' }}
-                topic={topic}
-              />
-            </Link>
-          ))}
-        </CardGrid>
-      </Box>
-    </Layout>
-  );
-};
+const Components = () => (
+  <Layout
+    backgroundImage={{
+      src: {
+        dark: '/carte-components-dark.svg',
+        light: '/carte-components-light.svg',
+      },
+      alt: 'HPE Carte Design System',
+      fit: 'contain',
+      margin: { top: '-200px', left: '-50px' },
+      small: {
+        margin: { top: '-250px', left: '0' },
+      },
```

Comment:
I'm not sure about those random numbers. can you back those numbers up?

Reply:
Trying to align as closely with the placement of the images as in the designs from Chris. T-shirt sizing doesn't give enough specificity to control how the images overflow of the edge of the page.